src/compiler/commandLineParser.ts

```diff
@@ -312,9 +312,19 @@ namespace ts {
             }
         },
         {
-            name: "traceModuleResolution",
+            name: "typesSearchPaths",

```

Comment:
We should mark this experimental as the semantics around this are still somewhat up for debate